store/test_helpers.go

```diff
@@ -0,0 +1,57 @@
+package store
+
+import (
+	"math/rand"
+
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	"github.com/rollkit/rollkit/types"
+)
+
+func getRandomBlock(height uint64, nTxs int) *types.Block {
```

Comment:
I think I've seen this multiple times as well, so include in the second refactor.